src/components/Cooking.js

```diff
@@ -0,0 +1,45 @@
+import Form from "./Form";
+import RecipeBook from "./RecipeBook";
+import { useState } from "react";
+
+const Cooking = () => {
+    const [newRecipes, setNewRecipes] = useState([]);
```

Comment:
I could be 100% wrong but confirm that useState([]) is supposed to have a square bracket in between? Could be your double click issues? ZERO idea lol ,just spitballin'